input/fsh/BFDR/profilesMigratingFromVRCL/ConditionPrepregnancyDiabetes_new.fsh

```diff
@@ -0,0 +1,20 @@
+Profile: ConditionPrepregnancyDiabetesNew
+Parent: USCoreConditionProblemsHealthConcernsProfile
+Title: "Condition - Prepregnancy Diabetes Vital Records"
+Description: "This Condition profile indicates that the mother had a pregnancy risk factor of prepregnancy diabetes."
+* ^meta.versionId = "2"
```

Comment:
remove meta info on this profile (and others - eclampsia hypertension, gestational diabetes, prepregnancy diabetes/hypertension, etc)